static/templates/admin/editStreamingPropertiesForm.html

```diff
@@ -2,14 +2,10 @@
 <form id="edit_streaming_properties" class="edit_form edit_streaming_properties_form">
     <div class="form_field">
         <label for="streaming_url"><span class="required">*</span> Streaming URL:</label>
-        <input id="streaming_url" type="text" name="streaming_url" value="<%= metadata.streamingUrl %>"/>
+        <input id="streaming_url" type="text" placeholder="Format: https://durastream.lib.unc.edu/..." name="streaming_url" value="<%= metadata.streamingUrl %>"/>
```

Comment:
The text box isn't all the wide, so it might make sense to do one or more of these: 
* remove "Format: " from the beginning of the placeholder text
* Move the text out of the placeholder and into an external message, like adding it to the required field text down below, then the user could potentially copy the base url (although i'm not sure if that would be useful or not since you need the property names, but with more space I guess you could show all the fields)
* Make the streaming_url input wider so that the user can see a whole url at once without needing to horizontally scroll